public/index.html

```diff
@@ -93,148 +93,169 @@
-                        <a class="thumbnail" href="#">
-                            <img id="nebula-22" class="img-responsive" src="" alt="">
-                        </a>
-                    </div>
-                    <div class="col-lg-3 col-centered thumb">
-                        <a class="thumbnail" href="#">
-                            <img id="nebula-23" class="img-responsive" src="" alt="">
-                        </a>
-                    </div>
-                    <div class="col-lg-3 col-centered thumb">
-                        <a class="thumbnail" href="#">
-                            <img id="nebula-24" class="img-responsive" src="" alt="">
-                        </a>
-                    </div>
-                </div>
-                <div class="row-centered">
-                    <div class="col-lg-3 col-centered thumb">
-                        <a class="thumbnail" href="#">
-                            <img id="nebula-19" class="img-responsive" src="" alt="">
-                        </a>
-                    </div>
-                    <div class="col-lg-3 col-centered thumb">
-                        <a class="thumbnail" href="#">
-                            <img id="nebula-6" class="img-responsive" src="" alt="">
-                        </a>
-                    </div>
-                    <div class="col-lg-3 col-centered thumb">
-                        <a class="thumbnail" href="#">
-                            <img id="nebula-7" class="img-responsive" src="" alt="">
-                        </a>
-                    </div>
-                    <div class="col-lg-3 col-centered thumb">
-                        <a class="thumbnail" href="#">
-                            <img id="nebula-8" class="img-responsive" src="" alt="">
-                        </a>
-                    </div>
-                    <div class="col-lg-3 col-centered thumb">
-                        <a class="thumbnail" href="#">
-                            <img id="nebula-9" class="img-responsive" src="" alt="">
-                        </a>
-                    </div>
+                <div class="col-lg-7">
+                    <div class="row plotbox" id="neighbourpca"></div>
                 </div>
-                <div class="row-centered">
-                    <div class="col-lg-3 col-centered thumb">
-                        <a class="thumbnail" href="#">
-                            <img id="nebula-18" class="img-responsive" src="" alt="">
-                        </a>
-                    </div>
-                    <div class="col-lg-3 col-centered thumb">
-                        <a class="thumbnail" href="#">
-                            <img id="nebula-5" class="img-responsive" src="" alt="">
-                        </a>
+                <div class="col-lg-5 gallery">
+                    <div class="row-centered">
+                        <div class="col-lg-3 col-centered selected-image-display">
```

Comment:
One option for noobs such as myself would be to put a comment on first use of a non-obvious bootstrap class in a file, that explains the class and its purpose. For example, there would have been no way for me to know that the classes needed to add to 12. Other class names, such as "selected-image-display", are nice and self-explanatory. =)

Could you add such a comment above somewhere?